README.rst

```diff
@@ -61,10 +65,51 @@ In Scrapy <1.0:
         'scrapy_fake_useragent.middleware.RetryUserAgentMiddleware': 401,
     }
 
+Enabling providers
+---------------------------
+
+Each provider is enabled individually, and used in the order they are defined.
+In case a provider fails execute (for instance, it can `happen <https://github.com/hellysmile/fake-useragent/issues/99>`__ to fake-useragent because of it's dependency
```

Comment:
I think you can avoid linking the fork with smth like https://stackoverflow.com/a/20829706/771848.

Reply:
So, you want to link to the issue of this repo instead of the repo of fake-useragent?

Reply:
I thought we wanted to point out that fake-useragent can be down, and that's why we have fallbacks.

Reply:
Oh, this is my bad, you are right, please ignore my comment.